python_modules/dagster/dagster/_core/storage/event_log/sql_event_log.py

```diff
@@ -1953,14 +1955,22 @@ def _fetch_partition_keys_for_partition_def(self, partitions_def_name: str) -> S
 
         return [cast(str, row[1]) for row in rows]
 
-    def get_dynamic_partitions(self, partitions_def_name: str) -> Sequence[str]:
-        """Get the list of partition keys for a partition definition."""
-        self._check_partitions_table()
-        return self._fetch_partition_keys_for_partition_def(partitions_def_name)
-
     def has_dynamic_partition(self, partitions_def_name: str, partition_key: str) -> bool:
         self._check_partitions_table()
-        return partition_key in self._fetch_partition_keys_for_partition_def(partitions_def_name)
+        query = (
+            db_select([1])
```

Comment:
nit: can we change this to be `db.select([DynamicPartitionsTable.c.partition])` just to be a bit more explicit about the selected column?

Reply:
Sure, done!